tap_stripe/schemas/charges.json

```diff
@@ -1,1776 +1 @@
-            },
-            "tokenization_method": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "address_country": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "fingerprint": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "address_city": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "last4": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "address_line2": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "customer": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "brand": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "country": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "dynamic_last4": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "name": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "funding": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "address_zip": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "address_zip_check": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "cvc_check": {
-              "type": [
-                "null",
-                "string"
-              ]
-            },
-            "type": {
-              "type": [
-                "null",
-                "string"
-              ]
-            }
-          }
-        },
-        "type": {
-          "type": [
-            "null",
-            "string"
-          ]
-        }
-      }
-    },
-    "balance_transaction": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "amount_refunded": {
-      "type": [
-        "null",
-        "integer"
-      ]
-    },
-    "failure_code": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "dispute": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "description": {
-      "type": [
-        "null",
-        "string"
-      ]
-    },
-    "updated": {
-      "type": [
-        "null",
-        "string"
-      ],
-      "format": "date-time"
-    }
-  }
-}
+{"$ref": "shared/charge.json#/"}
```

Comment:
Moved schema for charges into `shared` folder as it is being used in the schema of newly added stream `payment_intents`.